src/Form/Type/RedirectType.php

```diff
@@ -19,12 +19,27 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
         $builder
             ->add('source', TextType::class, [
                 'label' => 'setono_sylius_redirect.form.redirect.source',
+                'required' => true,
             ])
             ->add('destination', TextType::class, [
                 'label' => 'setono_sylius_redirect.form.redirect.destination',
+                'required' => true,
```

Comment:
Required is default, so no need to add it

Reply:
I think it's better to make it explicit in order not to depend too much on default behaviours. But indeed, not mandatory.